src/test/appFormValidators.test.js

```diff
@@ -0,0 +1,362 @@
+var expect = require("chai").expect;
+
+var AppFormValidators = require("../js/stores/validators/AppFormValidators");
+
+describe("App Form Validators", function () {
+
+  describe("expects", function () {
+
+    before(function() {
+      this.v = AppFormValidators;
+    })
+
+    describe("app id", function () {
+      it("is not an empty string", function () {
+        expect(this.v.appIdNotEmpty("notempty")).to.be.true;
+        expect(this.v.appIdNotEmpty("")).to.be.false;
+      });
+
+      it("has no white spaces", function () {
+        expect(this.v.appIdNoWhitespaces("nowhitespace")).to.be.true;
+        expect(this.v.appIdNoWhitespaces("has white spaces")).to.be.false;
+      });
+    });
+
+    describe("cpus", function () {
+      it("is not an empty string", function () {
+        expect(this.v.cpus("")).to.be.false;
+      });
+
+      it("looks like a number", function () {
+        expect(this.v.cpus("not a number 666")).to.be.false;
+        expect(this.v.cpus(0.1)).to.be.true;
+        expect(this.v.cpus(5)).to.be.true;
+        expect(this.v.cpus("0.0001")).to.be.true;
+        expect(this.v.cpus("2")).to.be.true;
+      });
+    });
+
+    describe("disk", function () {
+      it("is not an empty string", function () {
+        expect(this.v.disk("")).to.be.false;
+      });
+
+      it("looks like a number", function () {
+        expect(this.v.disk("")).to.be.false;
+        expect(this.v.disk("not a number 666")).to.be.false;
+        expect(this.v.disk(0.1)).to.be.true;
+        expect(this.v.disk(5)).to.be.true;
+        expect(this.v.disk("0.0001")).to.be.true;
+        expect(this.v.disk("2")).to.be.true;
```

Comment:
I'm OK with multiple `expect`s per test if they test similar things, but I think we should at least break tests into happy and unhappy path cases, or we risk missing potential inputs as happened below. In this case, how about one test for `accepts numbers` and one for `rejects non-numeric strings`?